Vladyslav_Gurevych/HW8/MINorMAX.py

```diff
@@ -0,0 +1,44 @@
+def own_max(data: list, amount_of_result=1):
+    """
+       This function takes arguments(list and amount_of_result) and returns a list with a max value.
+       The number of output values can be changed by entering amount_of_result.
+    """
+    result = []
+    temp = data[:]
+    for count in range(amount_of_result):
+        max_digit = temp[0]
+        for i in range(1, len(temp)):
+            if max_digit < temp[i]:
+                max_digit = temp[i]
+            else:
+                continue
+        result.append(max_digit)
+        temp.remove(max_digit)
+    return result
+
+
+def own_min(data: list, amount_of_result=1):
```

Comment:
annotations for amount_of_result.
amount_of_result: int = 1

and for return statment

Reply:
+